phoenix-tracing-webapp/src/main/java/org/apache/phoenix/tracingwebapp/http/Main.java

```diff
@@ -62,15 +63,18 @@ public int run(String[] arg0) throws Exception {
         final String home = getConf().get(TRACE_SERVER_HTTP_JETTY_HOME_KEY,
                 DEFAULT_HTTP_HOME);
         //setting up the embedded server
-        ProtectionDomain domain = Main.class.getProtectionDomain();
-        URL location = domain.getCodeSource().getLocation();
-        String webappDirLocation = location.toString().split("target")[0] +"src/main/webapp";
+        String webappDirLocation = DEFAULT_WEBAPP_DIR_LOCATION;
```

Comment:
Suggest dropping `webappDirLocation` and just using `DEFAULT_WEBAPP_DIR_LOCATION` since this is not configurable anyways.